templates/base.html

```diff
@@ -1,6 +1,18 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <style>
```

Comment:
Why isn't this part of the css file?

Reply:
I've done small research and read that it loads page faster (with correct font) :)